src/org/scada_lts/web/mvc/api/PointValueAPI.java

```diff
@@ -365,6 +367,11 @@ public ResponseEntity<String> getValue(@PathVariable("xid") String xid, HttpServ
             if (user != null) {
                 DataPointVO dpvo = dataPointService.getDataPoint(xid);
                 PointValueTime pvt = pointValueService.getLatestPointValue(dpvo.getId());
+                
+                // API should show datapoint is disabled if datasource is disabled
+                DataSourceVO<?> dataSourceVO = dataSourceService.getDataSource(dpvo.getDataSourceId());
+                dpvo.setEnabled(dpvo.isEnabled() && dataSourceVO.isEnabled());
```

Comment:
DataPointRT dp = Common.ctx.getRuntimeManager().getDataPoint(dpvo.getId());
dpvo.setEnabled(dp != null);
if dp is not null then data point running;